.gitignore

```diff
@@ -2,3 +2,6 @@ _build
 .ipynb_checkpoints
 *.bak
 .DS_Store
+
+article.pdf
+morganton2025.pdf
```

Comment:
I just put these in so we aren't repeatedly committing and replacing large PDFs. Once things are stable, we can remove these and commit the PDFs.